### PR TITLE
Add the schema, queries and data

### PR DESCRIPTION
**Pull request by idungstanley**
- [ ] I created a database named vet_clinic.
- [ ] I create a table named animals with the following columns:
  - id: integer
  - name: string
  - date_of_birth: date
  - escape_attempts: integer
  - neutered: boolean
  - weight_kg: decimal

- [ ] I inserted some data's into the table as rows
- [ ] I performed the following queries:

     - Find all animals whose name ends in "mon".
     image
     - List the name of all animals born between 2016 and 2019.
      image
     - List the name of all animals that are neutered and have less than 3 escape attempts.
     image
     - List the date of birth of all animals named either "Agumon" or "Pikachu".
      image
     - List name and escape attempts of animals that weigh more than 10.5kg
       image
     - Find all animals that are neutered.
        image
     - Find all animals not named Gabumon. 
        image
     - Find all animals with a weight between 10.4kg and 17.3kg (including the animals with the weights that equals precisely 10.4kg or 17.3kg)
        image

### CODE REVIEW

File: queries.sql [hunk 1]
```diff
@@ -1,3 +1,10 @@
 /*Queries that provide answers to the questions from all projects.*/
 
-SELECT * from animals WHERE name = 'Luna';
+SELECT * from animals WHERE name like = '%mon';
+SELECT * from animals WHERE date_of_birth BETWEEN DATE '2016-01-01' and '2019-12-31';
```
Comment (andersonlebon) [inline queries.sql]: - [x] Kindly note that the requirements of this project say that this query should only select names of all animals born between 2016 and 2019. only names not all columns.
image

File: queries.sql [hunk 2]
```diff
@@ -1,3 +1,10 @@
 /*Queries that provide answers to the questions from all projects.*/
 
-SELECT * from animals WHERE name = 'Luna';
+SELECT * from animals WHERE name like = '%mon';
+SELECT * from animals WHERE date_of_birth BETWEEN DATE '2016-01-01' and '2019-12-31'; 
+SELECT * from animals WHERE neutered is true and escape_attempts < 3;
```
Comment (andersonlebon) [inline queries.sql]: - [x] The same here kindly select only the names of the result query.

**Review by andersonlebon — CHANGES_REQUESTED** [review]
## Hi @idungstanley 

Good job so far! I know you have worked hard on making this milestone But  There are some issues that you still need to work on to go to the next project but you are almost there!

almost-there
## To highlight :champagne: 
- Good commit history :heavy_check_mark: 
- Good PR description :heavy_check_mark: 
## Required Changes ♻️

- [ ] Kindly check comments under the review.

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by Munsa1 — APPROVED** [review]
Hi @idungstanley, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights
:heavy_check_mark: Descriptive Readme
:heavy_check_mark: Passing Linters
:heavy_check_mark: Correct Gitflow

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**